test/functional/wallet_basic.py

```diff
@@ -496,6 +496,13 @@ def run_test(self):
         self.nodes[0].setlabel(change, 'foobar')
         assert_equal(self.nodes[0].getaddressinfo(change)['ischange'], False)
 
+        # Test "decoded" field value in gettransaction response
+        self.log.info("Testing gettransaction decoding...")
+        self.nodes[0].generate(1)
+        txid = self.nodes[0].sendtoaddress(self.nodes[0].getnewaddress(), 1)
+        tx = self.nodes[0].gettransaction(txid=txid, decode=True)
+        assert_equal(tx["decoded"], self.nodes[0].decoderawtransaction(tx["hex"]))
```

Comment:
Or you can simply change L488:
```diff
-tx = self.nodes[0].decoderawtransaction(self.nodes[0].gettransaction(txid)['hex'])
+tx = self.nodes[0].gettransaction(txid, decode=True)['decoded']
```

Reply:
Probably better to leave it as a separated test